Header header
bool success            # Did the last "say" command succeed
string said_text        # The text that was given as part of the "say" command  
